### ISSUE TITLE
speed-up loomio container start up time

### ISSUE THREAD

**Issue by OriHoch** [loomio/loomio-deploy#32]
currently the docker image builds all assets on start-up, should look into building assets as a docker build step or cache the build assets and rebuild only when necessary

**Comment by robguthrie** [comment]
Yes, unfortunately it is necessary to build assets on startup because we read ENV configuration to decide on colour theme etc.

I would like to explore other options.. but it is necessary to make Loomio themeable, so reading ENV before compiling assets seems to be the only way to do this. Otherwise I guess we need to use inline styles for theme colours, or override colours with a special generated stylesheet.
